selftests/checkall

```diff
@@ -5,6 +5,7 @@ run_rc() {
     eval $1
     if [ $? != 0 ]; then
         GR=1
+        [ ! "$SELF_CHECK_CONTINUOUS" ] && exit 1
```

Comment:
OK, I'll send it as new patch to be able to revert in case we don't want that behavior. (although I like it). Btw I'm wondering if anyone uses the AVOCADO selfcheck. It was a nice demonstration but I'd remove it...